2-mandatory/1-create-functions.js

```diff
@@ -3,17 +3,19 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
-}
+function first5(arr) {
+   return arr.slice(0,5);
+  }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+     return arr.slice().sort();
```

Comment:
Very nice way to create a copy of an array using slice().

Actually making copies when programming is not always as easy as one may think. If you want, you could have a look at this discussion about how to copy things in javascript:
https://www.freecodecamp.org/news/copying-stuff-in-javascript-how-to-differentiate-between-deep-and-shallow-copies-b6d8c1ef09cd/